code/datums/ai_law_sets.dm

```diff
@@ -25,7 +25,7 @@
 
 /******************** Nanotrasen/Malf ********************/
 /datum/ai_laws/nanotrasen
-	name = "NT Default"
+	name = "NT Recommended"
```

Comment:
@SamHPurp Yeah I can agree that this isn't the best rename. Any suggestions? Should I just name it "Nanotrasen"?